### ISSUE TITLE
Expand Jest moduleNameMapper to include common file types

### ISSUE THREAD

**Issue by gihrig** [mxstbr/react-boilerplate#1292]
The existing configuration informs Jest to avoid testing `.css` and `.jpg` files.

Other files types cause testing errors.

For example, changing

`import Banner from './banner.jpg';`
to
`import Banner from './banner.png'`

Ref: [components/Header/index.js](https://github.com/mxstbr/react-boilerplate/blob/dev/app/components/Header/index.js#L8)

Throws:
![image](https://cloud.githubusercontent.com/assets/1481063/20773497/f42522b4-b706-11e6-880e-a5d8f762e6b8.png)

The solution is to include common file types in Jest's `moduleNameMapper` key.

From:
````json
    "moduleNameMapper": {
      ".*\\.css$": "<rootDir>/mocks/cssModule.js",
      ".*\\.jpg$": "<rootDir>/mocks/image.js"
    },
````
To:
````json
    "moduleNameMapper": {
      ".*\\.(css|less|styl|scss|sass)$": "<rootDir>/mocks/cssModule.js",
      ".*\\.(jpg|jpeg|png|gif|eot|otf|webp|svg|ttf|woff|woff2|mp4|webm|wav|mp3|m4a|aac|oga)$": "<rootDir>/mocks/image.js"
    },
````
Ref: [Jest Webpack Tutorial](https://facebook.github.io/jest/docs/tutorial-webpack.html)

**Comment by philipchmalts** [comment]
+1

**Comment by gihrig** [comment]
Resolved by #1293
